doc/userguide/setting-up-ipsinline-for-linux.rst

```diff
@@ -347,3 +347,76 @@ The following snippet shows a possible :ref:`suricata-yaml-threading` configurat
         - worker-cpu-set:
             cpu: [ 2,4,6,8,10,12,14,16 ]
 
+Netmap IPS mode
+~~~~~~~~~~~~~~~
+
+Using Netmap to support IPS requires setting up pairs of interfaces; packets are received
+on one interface within the pair, inspected by Suricata, and transmitted on the other
+paired interface. You can use native or host mode; host mode is used when the interface
```

Comment:
think it would be good to briefly describe native vs host. Where do packets go, do we take ownership of the iface, etc

Reply:
Will do.